assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -3,6 +3,18 @@
     <span>Current log from {{start}}&nbsp;</span>
     <span ng-if="end">to {{end}}</span>
   </div>
+
+  <div row main-axis="end" ng-if="canViewArchive">
+    <form
+      ng-if="canViewArchive"
+      action="{{kibanaAuthUrl}}"
+      method="POST" target="_blank">
```

Comment:
The target causes this to open a new tab/window.  Is this the way the other apps (hawtio?) integrate?  I thought the desire was for a user to navigate to logs within the same window? @liggitt ?? @jwforres ??